agent/mibgroup/ip-mib/data_access/scalars_linux.c

```diff
@@ -139,7 +139,7 @@ netsnmp_arch_ip_scalars_ipDefaultTTL_set(u_long value)
 
     rc = fprintf(filep, "%ld", value);
     fclose(filep);
-    if (1 != rc) {
+    if (0 >= rc) {
```

Comment:
Please change '0 >= rc' into 'rc <= 0'. See also the Criticism section in https://en.wikipedia.org/wiki/Yoda_conditions.

Reply:
Done